src/mozanalysis/types.py

```diff
@@ -19,3 +19,8 @@ class Uplift(str, Enum):
 EstimatesByBranch = dict[BranchLabel, Estimates]
 
 CompareBranchesOutput = dict[ComparativeOption, EstimatesByBranch]
+
+
+class ExperimentalUnit(str, Enum):
```

Comment:
"Experimental unit" means both "randomization unit" and "analysis unit". Right now, our randomization unit (per-install, sub-profile level) differs from our analysis unit (profile level, keyed by client_id). One of the big advantages of switching to group ID is that that synchronizes the two units, which has some nice benefits (or rather, removes some weird downsides). 

Now, given what I said above, I suppose we _should_ call this the `AnalysisUnit`, because for legacy experiments this is only the analysis unit, not exactly the randomization unit. I actually had it that way, but [switched](https://github.com/mozilla/mozanalysis/pull/273/commits/380b00d6de0a2028f3421a05a8dfac59542005f1).  

Thoughts?

Reply:
Thanks for the clarification! I think in that case it makes sense to call it `AnalysisUnit`.

Reply:
Awesome, I'll update.